web-ui/src/pages/ReceivedRequestsPage.jsx

```diff
@@ -214,84 +284,167 @@ const ReceivedRequestsPage = () => {
           <div style={{ marginTop: "1em" }}>
-            {Array.from({ length: 1 })
-              .map((_, index) => <SkeletonLoader key={index} type="received_requests" />)
-            }
+            {Array.from({ length: 1 }).map((_, index) => (
+              <SkeletonLoader key={index} type="received_requests" />
+            ))}
           </div>
-        }
-        {!isLoading && <div style={{ marginTop: "1em" }} className="no-requests-message">
-          <Typography variant="body1">{submittedRequests.length} submitted request{submittedRequests.length === 1 ? "" : "s"} currently hidden</Typography>
-        </div>
-        }
+        )}
+        {!isLoading && (
+          <div style={{ marginTop: "1em" }} className="no-requests-message">
+            <Typography variant="body1">
+              {submittedRequests.length} submitted request
+              {submittedRequests.length === 1 ? "" : "s"} currently hidden
+            </Typography>
+          </div>
+        )}
       </Collapse>
       <Collapse in={submittedRequestsExpanded} timeout="auto" unmountOnExit>
-        {!isLoading &&
+        {!isLoading && (
           <div className="submitted-requests-container">
-            {submittedRequests.length === 0 &&
-              <div className="no-requests-message"><Typography variant="body1">No submitted feedback requests</Typography></div>
-            }
-            {submittedRequests.length > 0 && filteredSubmittedRequests.length === 0 &&
-              <div className="no-requests-message"><Typography variant="body1">No submitted feedback requests</Typography></div>
-            }
+            {submittedRequests.length === 0 && (
+              <div className="no-requests-message">
+                <Typography variant="body1">
+                  No submitted feedback requests
+                </Typography>
+              </div>
+            )}
+            {submittedRequests.length > 0 &&
+              filteredSubmittedRequests.length === 0 && (
+                <div className="no-requests-message">
+                  <Typography variant="body1">
+                    No submitted feedback requests
+                  </Typography>
+                </div>
+              )}
             {filteredSubmittedRequests.map((request) => (
               <ReceivedRequestCard key={request.id} request={request} />
             ))}
           </div>
-        }
+        )}
+      </Collapse>
+      <div className="request-section-header">
+        <Typography variant="h5">Canceled Requests</Typography>
+        <IconButton
+          onClick={() => setCanceledRequestsExpanded(!canceledRequestsExpanded)}
+          aria-label="show more"
+          className={
+            canceledRequestsExpanded ? classes.expandOpen : classes.expandClose
+          }
+          size="large"
+        >
+          <ExpandMoreIcon />
+        </IconButton>
+      </div>
+      <Collapse in={!canceledRequestsExpanded} timeout="auto" unmountOnExit>
```

Comment:
The other collapsable (for submitted) has a `Divider` above the line corresponding to this. You should probably make sure they match.